src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,13 @@
+@mixin calendar-day {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  width: $day-width;
```

Comment:
This mixin is useless, as calendar-day have unique styles and you can't reuse them, just put these styles under calendar day selector